src/compile-cache.js

```diff
@@ -16,7 +16,7 @@ var cacheDirectory = null
 exports.setAtomHomeDirectory = function (atomHome) {
   var cacheDir = path.join(atomHome, 'compile-cache')
   if (process.env.USER === 'root' && process.env.SUDO_USER && process.env.SUDO_USER !== process.env.USER) {
-    cacheDir = path.join(cacheDirectory, 'root')
+    cacheDir = path.join(atomHome, 'root')
```

Comment:
Thanks for this. I think before, we used `cacheDir` here, so that the compile-cache would be at `~/.atom/compile-cache/root`.